src/js/utils/formatNumber.jsx

```diff
@@ -0,0 +1,4 @@
+const formatNumber = (value, locale = 'en-US') =>
+  new Intl.NumberFormat(locale).format(value);
```

Comment:
Can we pull the current locale here? If the locale argument is not provided, we will use the one selected for the application.